### ISSUE TITLE
Bug: cannot convert undefined or null to object (v0.1.1)

### ISSUE THREAD

**Issue by favoyang** [ntnyq/vuepress-plugin-social-share#20]
It seems a bug that the plugin is not functional since 0.1.1. i.e. the global UI disappears and using social-share local tag raising component not found error. Fallback to 0.1.0 resolves the issue.

The only error I get is `warning Cannot convert undefined or null to object`.

```
$ vuepress build docs
wait Extracting site metadata...
tip Apply local theme at C:\Users\favoy\Documents\projects\openupm\openupm\docs\.vuepress\theme...
tip Apply theme local (extends @vuepress/theme-default) ...
warning Cannot convert undefined or null to object ##### THIS LINE #####
tip Apply plugin container (i.e. "vuepress-plugin-container") ...
tip Apply plugin @vuepress/last-updated (i.e. "@vuepress/plugin-last-updated") ...
tip Apply plugin @vuepress/register-components (i.e. "@vuepress/plugin-register-components") ...
...
```

My configuration file is here: https://github.com/openupm/openupm/blob/master/docs/.vuepress/config.js

**Comment by favoyang** [comment]
BTW, wonderful plugin.

**Comment by ntnyq** [comment]
Thank you for reporting.
This has been fixed in **v0.1.2**. Let me know if you still face issues.

**Comment by favoyang** [comment]
Thanks for the fix, it works for me.
